examples/from-scratch/README.md

```diff
@@ -26,8 +26,15 @@ After we have a Cloud Spanner database created, we'll need a few variables:
 * Instance name
 * Database name aka DisplayName
 
-Once in, please edit the file `hc/local_settings.py` and make the section `DATABASES` into the following:
+Note: `django_spanner` MUST be added as the first entry into `INSTALLED_APPS`.
+
+Once in, please edit the file `hc/local_settings.py` and make the sections `INSTALLED_APPS` and `DATABASES` into the following:
 ```python
+INSTALLED_APPS = [
+    'django_spanner',
```

Comment:
Rather than a separate note above, perhaps it would be more clear like this:

`'django_spanner',   # Must be listed first.`

Reply:
Ack, thanks!

Reply:
Moved around.